pom.xml

```
@@ -384,30 +384,28 @@
 
 
         <!-- ###  PAYARA  ### -->
-
         <profile>
-            <id>payara4</id>
+            <id>Payara Community</id>
             <activation>
                 <property>
                     <name>payara.version</name>
-                    <value>/4.*/</value>
+                    <value>/^[5-9]\.[0-9]{4}(-SNAPSHOT|\..*)*/</value>
```

It would never be 5.2020-SNAPSHOT - there will always be a release number.
Also I don't see the need to explicitly say it ends in snapshot.
```suggestion
                    <value>/^[5-9]\.[0-9]{4}\.[0-9].*/</value>
```